lms/static/sass/discussion/elements/_navigation.scss

```diff
@@ -14,15 +14,15 @@
 // Header
 // ------
 .forum-nav-header {
-  @include box-sizing(border-box);
+  box-sizing: border-box;
   display: table;
-  border-bottom: 1px solid $gray-l2;
+  border-bottom: 1px solid $forum-color-border;
   width: 100%;
   background-color: $gray-l3;
 }
 
 .forum-nav-browse {
-    @include box-sizing(border-box);
+    box-sizing: border-box;
     display: table-cell;
```

Comment:
See my note above. Using table layouts causes screenreaders to switch reading modes. If this isn't tabular data, we shouldn't be displaying it as such.